### PR TITLE
local man rebalances bulldog and SAW, ruins everything

### PR DESCRIPTION
**Pull request by MMMiracles**
:cl: MMMiracles
tweak: Bulldog now starts with stun-slugs instead of its usual 60-brute slugs. Slugs now cost 3 TC instead of 2.
tweak: Most SAW ammo has been slightly buffed in damage so maybe it'll be worth buying now along with a slight cost decrease for the gun itself.
tweak: Incen shells for the shotgun now apply extra firestacks on hit so it actually sets people on fire instead of turning them into a light show.
tweak: Incen rounds for the SAW now leave a fire trail similar to the bulldog's dragonsbreath round.
add: Nuke Ops now have the ability to purchase breaching shells, a weaker variant of the meteorslugs that can still push back people/airlocks/mechs/corgis.
fix: SAW should now use its in-hand sprites for an open-closed magazine
/:cl:

SAW's TC cost has been lowered to 18 TC as to give it some potential appeal over more compact weapons such as the m90.

Default starting magazine in the bulldog has been switched to the stun slugs, because starting with a literal free revolver is dumb-dumb and if I remember correctly, wasn't meant to be as initially powerful as it has become.

SAW damage values:
- Default
  - does 45 brute with 5 AP
- Bleeding
  -  does 20 brute with 35 bloodloss per hit
- AP
  - does 40 brute with 75 AP
- Hollow 
  - does 60 brute with -10 AP
- Incen
  - does 7 brute with 3 firestacks applied per hit 
  - leaves a fire trail similar to the bulldog's dragons breath

Breaching shell values:
- does 10 brute, 4 weaken, and 4 stun. works exactly as the meteorshot in terms of knocking shit out of the way

### CODE REVIEW

**Comment by Zilenan91** [comment]
Make the AP rounds have 100% AP so that they're worth buying in extreme circumstances (like events or something). Maybe nerf the damage to compensate.

**Comment by KorPhaeron** [comment]
>SAW damage values

Could you please give the SAW its own caliber so that the mosin balance stops changing with it?

**Comment by MMMiracles** [comment]
oh right i keep forgetting about that

**Comment by PKPenguin321** [comment]
i think the syntax for the changelog is "rcsadd" not "add"

**Comment by TheNightingale** [comment]
Weren't free Bulldogs removed anyway?

**Comment by KorPhaeron** [comment]
@TheNightingale 

They got removed, but replaced with TC to buy them with, and they now come with a free elite hardsuit, so the problem is actually worse

**Comment by Dorsisdwarf** [comment]
Finally someone fixes my balance fuckup woooo

**Comment by MMMiracles** [comment]
local man wonders why this has been up for over 4 days

**Comment by tkdrg** [comment]
Could you replace the elite hardsuits in the bulldog kit with some useless fluff item while you're at it?

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -385,7 +386,6 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "An alternative 8-round stun slug magazine for use with the Bulldog shotgun. \
 			Saying that they're completely non-lethal would be lying."
 	item = /obj/item/ammo_box/magazine/m12g/stun
-	cost = 3
```
Comment (tkdrg) [inline code/modules/uplink/uplink_item.dm]: Why remove the cost? Let people buy more ammo.

Comment (KorPhaeron) [inline code/modules/uplink/uplink_item.dm]: He added it to the parent at 381

Comment (tkdrg) [inline code/modules/uplink/uplink_item.dm]: it's not actually parent, it's two types with the same path
just rename one of them please

**Comment by WJohn** [comment]
The default ammo is in all cases strictly better than AP, as regular armor/helmets only have 15% bullet resistance. Even if it didn't have the AP, you'd still do 114.75 brute in 3 shots, with the AP you do 121.5. AP on the other hand could only ever do 105 brute since pretty much no armor exists on the station with good enough damage resistance, and it'd still be utterly worthless against someone with bulletproof armor since that offers something like 80% bullet resistance.

Please run these numbers through a calculator when rebalancing them, there's no reason to go for AP rounds.

**Comment by LanCartwright** [comment]
ill remove elite hardsuits from bundle tonight and replace it with something more appropriate.

**Comment by TheNightingale** [comment]
@LanCartwright The 'free' elite hardsuits aren't really free, since you pay for the bundle as a whole (which costs more than just a Bulldog). Only a standard Bulldog is free. If you're removing the elite hardsuit, maybe drop the bundle price to 14TC and throw in some thermals or an emag?

**Comment by tkdrg** [comment]
@MMMiracles please fix the thing i mentioned

**Comment by MMMiracles** [comment]
@tkdrg the pathname was fixed for stun slugs and shotgun ammo's parent already has the cost at 2 TC. I don't see why stun slugs would need to be 3 TC im.

**Comment by tkdrg** [comment]
yes i see i missed it before

**Pull request merged**